mediacheck.h

```diff
@@ -0,0 +1,60 @@
+#ifndef _MEDIACHECK_H
+#define _MEDIACHECK_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct mediacheck_digest_s mediacheck_digest_t;
+
+typedef int (* mediacheck_progress_t)(unsigned percent);
```

Comment:
would be nice to document this callback. percent parameter looks fine for me, but what is resulting int?

Reply:
explained in mediacheck_init()